Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -34,23 +34,25 @@ public function __construct(Container $container)
      * @param string $slot
      * @param View   $view
      * @param string $entity
+     * @param string $containerIds
      *
      * @return string
      */
-    public function renderNewForm($form, $widget, $slot, View $view, $quantum = null, $entity = null)
+    public function renderNewForm($form, $widget, $slot, View $view, $quantum = null, $entity = null, $containerIds = null)
```

Comment:
I wonder if it's normal you don't pass the $containerIds to the renderNewForm function here : https://github.com/jdubuisson/victoire/blob/cc2dc679c6ed033d1317a04425bfcebbe56fc067/Bundle/WidgetBundle/Builder/WidgetFormBuilder.php#L106 ?